packages/api/src/user/user.ts

```diff
@@ -73,10 +78,180 @@ export const createPlatformUser = async (
 ) => {
   const response = await client.call<Response<ExternalUserResponseData>>({
     url: `/v1/externalUsers`,
-    method: 'post',
+    method: 'POST',
     body: { userType, externalUserID: externalUserId },
     authorization: true,
   })
 
   return response.data
 }
+
+export const getUserMasqueradeToken = async (
+  client: Client,
+  { userId }: GetUserArgs,
```

Comment:
We tend to keep args & data types per function. It makes changes smoother. I suggest to create a new GetUserMasqueradeTokenArgs type.

Reply:
🔝